test/integration/bulk-load-test.js

```diff
@@ -351,7 +351,7 @@ describe('Bulk Load Tests', function() {
   });
 
   it('should test stream bulk load', function(done) {
-    this.timeout(50000);
+    this.timeout(200000);
```

Comment:
change this because of that this test always gives me a time out error. Increase this make the test pass.